boards/arm/sam_v71_xult/doc/index.rst

```diff
@@ -0,0 +1,181 @@
+- Supercap backup
+- AT24MAC402 EEPROM
+- IS42S16100E 16 Mb SDRAM
+- S25FL116K 16 Mb QSPI
+- WM8904 low power stereo audio codec
+- ATA6561 CAN Transceiver
+- SD card connector with SDIO support
+- Camera interface connector
+- MediaLB connector
+- Ethernet port
+- Micro-AB USB device
+- Micro-AB USB debug interface supporting CMSIS-DAP, Virtual COM Port and Data
+  Gateway Interface (DGI)
+- JTAG interface connector
+- One reset and two user pushbutton
+- Two yellow user LED
+
+Supported Features
+==================
+
+The sam_v71_xplained_ultra board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| USART     | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| ETHERNET  | on-chip    | ethernet                            |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | watchdog                            |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| ADC       | on-chip    | ADC via AFEC                        |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | USB device                          |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by Zephyr.
+
+The default configuration can be found in the Kconfig
+:zephyr_file:`boards/arm/sam_v71_xult/sam_v71_xult_defconfig`.
+
+Connections and IOs
+===================
+
+The `SAMV71-XULT User Guide`_ has detailed information about board
+connections.
+
+System Clock
+============
+
+The SAM V71 MCU is configured to use the 12 MHz external oscillator on the
+board with the on-chip PLL to generate a 300 MHz system clock.
+
+Serial Port
+===========
+
+The ATSAMV71Q21 MCU has five UARTs and three USARTs. One of the USARTs is
```

Comment:
Document which one is configured for the console here:  ``USART1 is configured ...``